datafusion-cli/src/command.rs

```diff
@@ -72,6 +76,22 @@ impl Command {
                     .print_batches(&batches, now)
                     .map_err(|e| DataFusionError::Execution(e.to_string()))
             }
+            Self::Include(filename) => {
+                if let Some(filename) = filename {
+                    let file = match File::open(filename) {
+                        Ok(file) => file,
+                        Err(error) => {
+                            return Err(DataFusionError::Execution(error.to_string()))
+                        }
```

Comment:
Yeah, something like this perhaps?

```suggestion
                    let file =  File::open(filename)
                        .map_err(|e| {
                            DataFusionError::Execution(format!("Error opening {:?} {}", filename, e))
                        })?;                  
```

As the `e` doesn't contain the filename

This gives something like: 

```sql
❯ \i /tmp/foo.dd
Execution error: Error opening "/tmp/foo.dd" No such file or directory (os error 2)
```

Reply:
thanks! yup of course, happy to add the documentation change as well :)